sample/app/src/test/java/com/squareup/anvil/sample/UserComponentTest.kt

```diff
@@ -0,0 +1,15 @@
+package com.squareup.anvil.sample
+
+import org.junit.Test
+
+class UserComponentTest {
+
+  @Test fun `UserComponent is generated`() {
+    val parent = DaggerAppComponent.create() as UserComponent.Parent
+    val baseComponent = parent.user().create()
+    val userComponent = baseComponent as UserComponent
+
+    assert(baseComponent.description() == UserDescriptionModule.provideDescription())
+    assert(userComponent.username() == UserDescriptionModule.provideName())
```

Comment:
I'll fix this after merging but in the future please use `assertThat` test helpers like the rest of the tests

Reply:
🙏